frontend/src/components/shared/Icon/Icon.js

```diff
@@ -1,13 +1,13 @@
-import React from "react";
-import { IconContainer, IconBold, IconSub } from "./IconStyles";
-
-const Icon = ({ setSwitchBanner = { setSwitchBanner } }) => {
-  return (
-    <IconContainer onClick={() => setSwitchBanner(false)}>
-      <IconBold>netlix</IconBold>
-      <IconSub>roulette</IconSub>
-    </IconContainer>
-  );
-};
-
-export default Icon;
+import React from "react";
+import { IconContainer, IconBold, IconSub } from "./IconStyles";
+
+const Icon = ({ setSwitchBanner = { setSwitchBanner } }) => {
+  return (
+    <IconContainer onClick={() => setSwitchBanner(false)}>
```

Comment:
again, same situation as with map callback would happen. On every render we'd have new onClick. Use useCallback